### ISSUE TITLE
Your operating system not supported

### ISSUE THREAD

**Issue by mooleshacat** [redis/RedisDesktopManager#4363]
**Describe the bug**
Your operating system is not supported!

**To Reproduce**
1. clone repo
2. cd src
3. ./configure
4. "Your operating system is not supported!"

**Expected behavior**
Configure to run

**Environment (please complete the following information):**
 - OS & version: Debian 9

**Additional context**
Worked in 0.9.8 now tells me to update to 0.9.9 but when compiling 0.9.9 it says "Your operating system is not supported!" which is obviously not true as I have older version installed already

suggest not making frequent updates if updates are going to break like this

**Comment by uglide** [comment]
Hello

Maybe you were on Ubuntu, Mint, etc before? Because Debian is not supported by `configure` script in any version of 0.9 branch.

**Comment by mooleshacat** [comment]
Nope.

I literally meant what I said. I installed it on this very computer, Debian, and I have not reinstalled it since, only the program has new version and Debian must be taken out now because I can't install it on the very computer I installed it on.

This is simple and straightforward. I installed RDM on Debian, a new version came out, I tried to install it but it does not work, and I am stuck on the previous version (currently running on Debian)

v0.9.8 on Debian 9 (this is sad I have to waste time explaining already whats explained)
![untitled](https://user-images.githubusercontent.com/43627985/50408283-fad13380-07b5-11e9-99c3-0499d3227f3f.png)
